### PR TITLE
Onvista PDF Extractor: add Dividend with Outbond Delivery Transaction

### PR DESCRIPTION
**Pull request by ZfT2**
Hello again :) here an update for the Onvista PDF Extractor.

### CODE REVIEW

**Comment by Nirus2000** [comment]
Hello @ZfT2 
... welcome back...
Please check the Contributing rules for PDF-Importers.

Alex

**Comment by buchen** [comment]
Hi @ZfT2, I will have a look at it. Is it guaranteed that there is always one account transaction in the document? Otherwise you might run into a NPE.

**Comment by ZfT2** [comment]
I'm not sure what Onvista will do in future, but for now all documents of this type have (only) one account transaction, yes.

File: name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/onvista/OnvistaPDFExtractorTest.java [hunk 1]
```diff
@@ -2799,6 +2799,62 @@ public void testDividendeWithReinvest01()
...
+        // check dividends transaction
+        AccountTransaction transaction = (AccountTransaction) results.stream().filter(TransactionItem.class::isInstance)
+                        .findFirst().orElseThrow(IllegalArgumentException::new).getSubject();
+
+        assertThat(transaction.getType(), is(AccountTransaction.Type.DIVIDENDS));
+
+        assertThat(transaction.getDateTime(), is(LocalDateTime.parse("2022-05-25T00:00")));
+        assertThat(transaction.getShares(), is(Values.Share.factorize(6)));
+        assertThat(transaction.getSource(), is("DividendeWithOutbondDelivery01.txt"));
+
```
Comment (Nirus2000) [inline name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/onvista/OnvistaPDFExtractorTest.java:2832]: Add here `assertThat(entry.getNote(), is("...bla bla..."));`

**Comment by Nirus2000** [comment]
Hello @ZfT2 
please have a look at the [Contributing Source](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md#pdf-importer) and if possible please try to use it. (e.g. addDeliveryInOutBoundTransaction)

I think we don't need [this](https://github.com/buchen/portfolio/pull/3204/commits/aff9b42ce76b5604a11fb9b3b9bdac215585e15f#diff-40278a441d1a75c703c0eff63b60c497961869cbc5969f712d7e6473e6633734R101-R112) customization.

Regards
Alex

**Comment by ZfT2** [comment]
OK, I  refactored it and removed the extra method to create the (unnecessary) outbond delivery block

**Comment by Nirus2000** [comment]
Perfect !! 👍🏻 💯

**Pull request merged**